pkg/api/repositoryRpm.go

```diff
@@ -0,0 +1,60 @@
+package api
+
+import (
+	"github.com/content-services/content-sources-backend/pkg/models"
+	"github.com/openlyinc/pointy"
+)
+
+type RepositoryRpm struct {
+	// RPM id
+	UUID string `json:"uuid"`
+	// The rpm package name
+	Name string `json:"name"`
+	// The architecture that this package belong to
+	Arch string `json:"arch"`
+	// The version for this package
+	Version string `json:"version"`
+	// The release for this package
+	Release string `json:"release"`
+	// Epoch is a way to define weighted dependencies based
+	// on version numbers. It's default value is 0 and this
+	// is assumed if an Epoch directive is not listed in the RPM SPEC file.
+	// https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/packaging_and_distributing_software/advanced-topics#packaging-epoch_epoch-scriplets-and-triggers
+	Epoch       *int32 `json:"epoch"`
```

Comment:
any reason to not just use 'int'?   also, why is this a pointer?

Reply:
Thanks! I miss this! I can see that into librpm an int32 is used here: https://ftp.osuosl.org/pub/rpm/api/4.4.2.2/parsePreamble_8c-source.html ; in the other hand, the pointer is because initially I set the field as default to null into the database (but I have checked it out and it is not in that way into the migration, so I have to remove the pointer).  I will change `*int32` by `int32`.

Reply:
Updated